develop-dotnet-plugins.md

```diff
@@ -1,52 +1,52 @@
-Flow is written in C#, so plugins written in .Net platform can directly communicate with Flow without extra protocols.
+Flow is written in C#, so plugins written in .NET platform can directly communicate with Flow without extra protocols.
 
 ## Initialization
 
 For C# Plugins, We recommend you use the [dotnet template](https://github.com/Flow-Launcher/dotnet-template) to generate a plugin template.
 
-In order to be recognized as a Flow DotNet plugin, the directory needs to have at least two files
-1. [`plugin.json`](plugin.json.md)
-2. A Dotnet Assembly that implements **[IPlugin](API-Reference/Flow.Launcher.Plugin/iplugin.md)** or **[IAsyncPlugin](API-Reference/Flow.Launcher.Plugin/iasyncplugin.md)** (remember to reference [Flow.Launcher.Plugin](https://www.nuget.org/packages/Flow.Launcher.Plugin/) by Nuget). The plugin template will add the reference and create  a `Main.cs` that implements `IPlugin`.
+To be recognized as a Flow DotNet plugin, the directory needs to have at least two files
+1. [`plugin.json`](/plugin.json.md)
+2. A Dotnet Assembly that implements **[IPlugin](/API-Reference/Flow.Launcher.Plugin/IPlugin.md)** or **[IAsyncPlugin](/API-Reference/Flow.Launcher.Plugin/IAsyncPlugin.md)** (remember to reference [Flow.Launcher.Plugin](https://www.nuget.org/packages/Flow.Launcher.Plugin/) by Nuget). The plugin template will add the reference and create  a `Main.cs` that implements `IPlugin`.
```

Comment:
Due to case-sensitivity, the links to the auto-generated docs were leading to 404.